cfpb/django-college-costs-comparison
# Add jquery and format markup

- Add jquery to templates to accommodate update in https://github.com/cfpb/cfgov-refresh/pull/4619
- Convert tabs to spaces in templates.
- Auto-indent markup in templates.
- Add more details to readme install section.

## Test
- `pip install -e ../django-college-costs-comparison`
- Visit http://localhost:8000/paying-for-college/manage-your-college-money/
- Open the dev console and see that there are no `$` or `jQuery` undefined errors.
- Click through the app and check the same.
